src/django-app/django-project/config/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path('auth/', include('djoser.urls')),
     path('auth/', include('djoser.urls.jwt')),
+    path('', include('shops.urls')),
```

Comment:
Let's add a prefix to the path. Something like `shops/`

Reply:
Ah, the way I've done the endpoints there IS a prefix, it's just specified [here](https://github.com/HKanwal/kapstone/blob/719856e3f5c0d7712a416b5c7b23ab5ff33c8863/src/django-app/django-project/shops/urls.py#L5-L11) instead.

Reply:
Ah wait, I think I get what you mean now. I will update this.